### DISCUSSION TITLE
Thanks to all developping this addon/ docker

### DISCUSSION THREAD

**Discussion by canedje** [matt8707/ha-fusion#127] [General]
I like to thank everybody developping this addon/ docker.
I'm very happy with it.
It is easy to configure and fast in usage at my walltablet.
It's logic there are improvements needed, but the developping schedule is very fast
Thanks!!

**Comment by Breakingx** [comment]
Just throwing my two cents in there, too.

It's incredible what Matt and all contributers are doing. I'm way to novice to add something to the project but I'm glad for every update it gets. Currently the functions of the sidebar and main view are limited but with what you can already do with it, it's already so pleasing to look at.

Thanks again!

**Comment by jaburges** [comment]
@canedje you mentioned you are running the ha-fusion docker on Unraid in another thread (I am as well). How are you running add-ons? Are you running HA as a VM?

**Comment by canedje** [comment]
Yes I do. But I do use severall dockers outside HA.
Nodered 
Z2m
Mosquito
EspHome
Go2rtc
